lib/src/showcase_widget.dart

```diff
@@ -149,10 +154,16 @@ class ShowCaseWidgetState extends State<ShowCaseWidget> {
     enableAutoPlayLock = widget.enableAutoPlayLock;
     enableAutoScroll = widget.enableAutoScroll;
     disableBarrierInteraction = widget.disableBarrierInteraction;
+    enableShowcase = widget.enableShowcase;
   }
 
   /// Starts Showcase view from the beginning of specified list of widget ids.
+  /// If this function is used when showcase has been disabled then it will
+  /// throw an exception.
   void startShowCase(List<GlobalKey> widgetIds) {
+    if (!enableShowcase) {
+      throw Exception("Trying show Showcase when it has been disabled");
```

Comment:
please write the complete sentence "You are trying to start Showcase while it has been disabled with enableShowcase parameter  to false from ShowCaseWidget"